### DISCUSSION TITLE
Fading out progression achievements on game page

### DISCUSSION THREAD

**Discussion by Nepiki** [RetroAchievements/RAWeb#2255] [Ideas]
Hi! I believe the topic of spoilering achievement have been mentioned before (although I couldn't find a Github/Site Feedback discussion right away), but I wanted to give an additional idea that may be beneficial.

My idea is to have an additional filter just like ''showing only missables'', and instead for fading out progression/win condition achievements. The reason I opted for fading them out rather than hiding them in a similar manner to what the other filter does, is that it can be very helpful for sets that are chronologically ordered--so most RPGs. This way, it hides the spoilery achievements and at the same time it has the additional benefit that you can make sure if you have gotten every achievement before a new story achievement unlocks, e.g. for missable achievements. I've actually demonstrated this in a few RAGuides I've made, such as [here](https://github.com/RetroAchievements/guides/wiki/Legend-of-Heroes,-The:-A-Tear-of-Vermillion-(PlayStation-Portable)). The player knows how many story-related achievements are hidden from them, so they also know how many achievements they can get before then.

Obviously it's a filter that mostly benefits RPGs and other story-heavy games, but I figured it would be beneficial to share this idea in case fading out content may not only have a function here but elsewhere too :)

![ProgressionFilter](https://github.com/RetroAchievements/RAWeb/assets/76693803/11625802-004c-4d29-a14c-ba78100453f7)

**Comment by Jamiras** [comment]
Spoiler achievements are discussed in #756.

I'd argue that this functionality would be more confusing than useful. If you see non-progression achievements without the context of the progression achievements, how will you know when they're applicable?

**Reply by Nepiki** [reply]
You do make a good point and perhaps my bias has influenced this suggestion there. I would say that having the context is a bonus rather than the end goal, but if spoilers are implemented in a way like https://github.com/RetroAchievements/RAWeb/discussions/756 then it indeed becomes redundant to have a blur or even a filter 😅
